include/libopencm3/stm32/f1/rcc.h

```diff
@@ -233,18 +233,19 @@
 /**@}*/
 
 /* SWS: System clock switch status */
-#define RCC_CFGR_SWS_SYSCLKSEL_HSICLK		0x0
-#define RCC_CFGR_SWS_SYSCLKSEL_HSECLK		0x1
-#define RCC_CFGR_SWS_SYSCLKSEL_PLLCLK		0x2
+#define RCC_CFGR_SWS_HSI			(0 << RCC_CFGR_SWS_SHIFT)
+#define RCC_CFGR_SWS_HSE			(1 << RCC_CFGR_SWS_SHIFT)
+#define RCC_CFGR_SWS_PLL			(2 << RCC_CFGR_SWS_SHIFT)
 
 /* SW: System clock switch */
 /** @defgroup rcc_cfgr_scs RCC_CFGR System Clock Selection
 @ingroup STM32F1xx_rcc_defines
 
 @{*/
-#define RCC_CFGR_SW_SYSCLKSEL_HSICLK		0x0
-#define RCC_CFGR_SW_SYSCLKSEL_HSECLK		0x1
-#define RCC_CFGR_SW_SYSCLKSEL_PLLCLK		0x2
+#define RCC_CFGR_SW_HSI				(0 << RCC_CFGR_SW_SHIFT)
+#define RCC_CFGR_SW_HSE				(1 << RCC_CFGR_SW_SHIFT)
+#define RCC_CFGR_SW_PLL				(2 << RCC_CFGR_SW_SHIFT)
+
```

Comment:
I generally prefer to keep these defines unshifted.  You've got the PLL factors unshifted, the original bits here were unshifted, and you've changed all of them to be shifted, is there any reason I can't see for that?

Reply:
(To clarify, my general rule is that single bits are defined shifted, but fields, _especially_ when the field value is relevant, should all be unshiftted)

Reply:
I just followed the current F0 code.

But of course I can unshift them, including F0.

Reply:
Yes, I'd rather you copied the f1 code on this, where it's unshifted.